.github/workflows/ci.yml

```diff
@@ -31,7 +31,8 @@ jobs:
         run: |
           pip install --no-deps .
           docker-compose up -d
-          pytest -nauto --dist loadfile -rs
+          sleep 10s
```

Comment:
Should we instead wait for the port to come up? Timed sleeps tend to be annoyingly flaky.

Reply:
Probably should, fixed it.